nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -1047,6 +1047,24 @@ int ddxProcessArgument(int argc, char *argv[], int i)
     return 0;
   }
 
+  /*
+   * Disable Xinerama (i.e. fake it in Screen.c) if somehow Xinerama support
+   * has been disabled on the cmdline.
+   */
+#if defined(PANORAMIX) && defined(RANDR)
+  if ((noPanoramiXExtension || PanoramiXExtensionDisabledHack) && noRRXineramaExtension)
+    nxagentChangeOption(Xinerama, 0);
+#else
+#ifdef RANDR
+  if (noRRXineramaExtension)
+    nxagentChangeOption(Xinerama, 0);
+#endif
+#ifdef PANORAMIX
+  if (noPanoramiXExtension || PanoramiXExtensionDisabledHack)
+    nxagentChangeOption(Xinerama, 0);
+#endif
+#endif
+
   return 0;
```

Comment:
This looks complicated.  I think it can be solved easier. At least the last ifdef kan be replaced by an else.